stats/stats_test.go

```diff
@@ -90,6 +100,17 @@ func TestTaggedStats_Timing(t *testing.T) {
 	m.AssertExpectations(t)
 }
 
+func TestTaggedStats_Close(t *testing.T) {
+	m := new(MockStats)
+	m.On("Close").Return(nil)
+
+	s := stats.NewTaggedStats(m, map[string]string{"global": "foobar"})
+	s.Close()
```

Comment:
Check the error and have an error case check

Reply:
The same as above.